packages/next/src/build/utils.ts

```diff
@@ -2049,13 +2049,6 @@ const dir = path.join(__dirname)
 process.env.NODE_ENV = 'production'
 process.chdir(__dirname)
 
-// Make sure commands gracefully respect termination signals (e.g. from Docker)
-// Allow the graceful termination to be manually configurable
-if (!process.env.NEXT_MANUAL_SIG_HANDLE) {
```

Comment:
removing this one is necessary for standalone mode to gracefully shutdown. The env is still in `start-server`